File: lab06/echon.py
#!/usr/bin/python
import re, sys
if((len(sys.argv)-1) != 2):
	print "Usage: ./echon.py <number of lines> <string>"
elif(sys.argv[1].isdigit() == False):
	print "./echon.py: argument 1 must be a non-negative integer"
elif(sys.argv[1] < 0):
	print "/echon.py: argument 1 must be a non-negative integer"
else:
	for x in range(int(sys.argv[1])):
    		print sys.argv[2]

File: lab05/echon.pl
#!/usr/bin/perl -w
if( $#ARGV != 1 ){
     print "Usage: ./echon.pl <number of lines> <string>\n"
}elsif( $ARGV[0] =~ /^[0-9]+$/ ) {
    for($i = 0; $i < $ARGV[0]; $i++) {
        print "$ARGV[1]\n";
    }
} else {
    print "./echon.pl: argument 1 must be a non-negative integer\n"
}
#    $var = "hello bob";
#    if ($var =~ /$pattern/) {
        # EXECUTES IF PATTERN MATCHES
#    }
